source/Service/MessageService.ts

```diff
@@ -100,6 +101,21 @@ export class MessageService
                 this.cooldownService.addCooldown(message.member, "XP", 60);
         }
     }
+    
+    private async addToTotalPings(user: User, message:Message)
+    {
+        let taggedMember = message.mentions.members.first();
+
+        if(taggedMember != message.member)
+        {
+            if(taggedMember)
+            {
+                let pingedMember: User = await this.userService.getUser(message.mentions.members.first(), message.guild);
+                pingedMember.totalPings ++;
+                pingedMember.save();
```

Comment:
this only takes care of the first user thats being pinged but we want to take care of all users being pinged